app/services/foreman_openscap/client_config/ansible.rb

```diff
@@ -0,0 +1,33 @@
+module ForemanOpenscap
+  module ClientConfig
+    class Ansible < Base
+      delegate :ansible_role_name, :to => :constants
+
+      alias config_item_name ansible_role_name
+
+      def type
+        :ansible
+      end
+
+      def available?
+        defined?(ForemanAnsible)
+      end
+
+      def help_text
+        "Make sure you have Ansible plugin installed and #{ansible_role_name} Ansible role imported."
```

Comment:
if this could include html, it would be great to link to import roles page (the same for puppet classes)